artifacthub/artifacthub-repo.yml

```diff
@@ -0,0 +1,15 @@
+# Artifact Hub repository metadata file
+#
+# Some settings like the verified publisher flag or the ignored packages won't
+# be applied until the next time the repository is processed. Please keep in
+# mind that the repository won't be processed if it has not changed since the
+# last time it was processed. Depending on the repository kind, this is checked
+# in a different way. For Helm http based repositories, we consider it has
+# changed if the `index.yaml` file changes. For git based repositories, it does
+# when the hash of the last commit in the branch you set up changes. This does
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+repositoryID: The ID of the Artifact Hub repository where the packages will be published to (optional, but it enables verified publisher)
+owners: # (optional, used to claim repository ownership)
+  - name: Lee Calcote
```

Comment:
I wonder if we might be able to use a mailing list like "developers@meshery.io" or if we need named, individual maintainers. Do you know if there are any examples of mailing lists being used, @ctcarrier?

Reply:
This page has some information on ownership claims which is the purpose of this section:
https://artifacthub.io/docs/topics/repositories/#ownership-claim

Pertinent to your question I think is: `The user requesting the ownership claim must appear in the list of owners in the metadata file, and the email listed must match with the one used to sign in in Artifact Hub. This information will be used during the process to verify that the requesting user actually owns the repository.`

So I think what matters is that the owner email address matches the email used to login to the AH control panel for Meshery. So maybe since this is early in the process an AH account for an email list could be created to make sure this isn't tied to an individual's email.